052-connector-state.md

```diff
@@ -0,0 +1,55 @@
+<!-- This template is provided as an example with sections you may wish to comment on with respect to your proposal. Add or remove sections as required to best articulate the proposal. -->
+
+# Support stopping connectors
+
+This proposal aims at adding support for stopping Kafka Connect connectors.
+
+## Current situation
+
+At the moment Strimzi allows pausing connectors. This is done by a property, `pause`, in the `KafkaConnectorSpec` schema. Whenever it is set to `true`, the connectors and its tasks are paused.
+
+For example:
+```yaml
+spec:
+  class: org.apache.kafka.connect.file.FileStreamSourceConnector
+  config:
+    file: /opt/kafka/LICENSE
+    topic: my-topic
+  pause: true
+  tasksMax: 1
+```
+
+## Motivation
+
+Since Kafka 3.5, Kafka Connect connectors can also be stopped. This is done by using a new REST endpoint `PUT /connectors/{connector}/stop`. Compared to paused where the connector and tasks stay instantiated, when stopped only the configuration of the connector is kept but nothing is actually running.
+
+A connector paused still uses some resources but it's faster to resume. So using pause is great if you want to halt data flowing for a short duration. For longer durations it's interesting to fully stop the connector with the new API. Also the stopped state will be required to use the offset delete/reset endpoints that should come with Kafka 3.6. For these reasons, Strimzi should support both pausing and stopping connectors.
```

Comment:
I think we need slight more justification than "interesting". (I'm picking on your choice of words; I don't doubt the usefulness of the feature).
```suggestion
A paused connector still uses some resources but it's faster to resume. So using pause is well-suited if you want to halt data flowing for a short duration. For longer durations it could be beneficial to free up memory and other resources by fully stopping the connector with the new API. The stopped state will be required to use the offset delete/reset endpoints that should come with Kafka 3.6. For these reasons, Strimzi should support both pausing and stopping connectors.
```